tests/constraints/valid-date-time.test.js

```diff
@@ -1,6 +1,6 @@
 import test from "ava";
 import { Literal } from "rdflib";
-import validDateTime from "./valid-date-time.js";
+import validDateTime from "../../src/constraints/valid-date-time.js";
```

Comment:
Should we import from the src/index.js file instead to match the "public" api more closely? The exports config prevents consumers from importing utils from their subpaths, so this path wouldn't work outside of this package.